src/utilities/_animations.scss

```diff
@@ -0,0 +1,185 @@
+/* Ripple */
+
+.ripple {
+  overflow: hidden;
+  position: relative;
+  transform: translate3d(0, 0, 0);
+
+  &::after {
+    background-image: radial-gradient(circle, #000 10%, transparent 10.01%);
```

Comment:
For this line, is there an existing color variable we can use instead of the hard coded `#000`?